src/wui/wui_request_parser.c

```diff
@@ -85,8 +100,7 @@ HTTPC_COMMAND_STATUS parse_http_reply(char *reply, uint32_t reply_len, httpc_hea
     return cmd_status;
 }
 
-uint32_t httpc_json_parser(char *json, uint32_t len, char *cmd_str) {
-    uint32_t ret_code = 1; // success is 0
+HTTPC_COMMAND_STATUS httpc_json_parser(char *json, uint32_t len) {
     int ret;
```

Comment:
if not necessary not use "int"  but uint32_t

Reply:
jsmn_parse() from jsmn.h returnes int.. That is why I use it there